docker/test/stateless_unbundled/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# docker build -t yandex/clickhouse-stateless-unbundled-test .
+FROM yandex/clickhouse-test-base
```

Comment:
Why just don't inherit it from `yandex/clickhouse-unbundled-builder`?

Reply:
There is problem with Ubuntu version. Stateless test needs for example `python-requests` and there is only `python3-requests` available in 20.04.
I just wanted to update packager and unbundled build first and leave stateless tests update to py3/20.04 later.